index.html

```diff
@@ -23,214 +23,242 @@
-                        <li class="work-language">HTML</li>
-                        <li class="work-language">Bootstrap</li>
-                        <li class="work-language">Ruby</li>
-                    </ul>
-                    <a href="#" class="btn see-work">See project</a>
+                <!-- WORK BOX (end) -->
+                <!-- WORK BOX (begin) -->
+                <div class="work-box w-disable">
+                    <div class="work-content">
+                        <h3 class="work-title">Profesional Art Printing Data</h3>
+                        <p class="work-description">
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. has been the industry's standard.
+                        </p>
+                        <ul class="work-languages-list">
+                            <li class="work-language">HTML</li>
+                            <li class="work-language">Bootstrap</li>
+                            <li class="work-language">Ruby</li>
+                        </ul>
+                        <a href="#" class="btn see-work">See project</a>
+                    </div>
                 </div>
-            </div>
-            <!-- WORK BOX (end) -->
-            <!-- WORK BOX (begin) -->
-            <div class="work-box w-disable">
-                <div class="work-content">
-                    <h3 class="work-title">Profesional Art Printing Data</h3>
-                    <p class="work-description">
-                        A daily selection of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry's standard.
-                    </p>
-                    <ul class="work-languages-list">
-                        <li class="work-language">HTML</li>
-                        <li class="work-language">Bootstrap</li>
-                        <li class="work-language">Ruby</li>
-                    </ul>
-                    <a href="#" class="btn see-work">See project</a>
+                <!-- WORK BOX (end) -->
+                <!-- WORK BOX (begin) -->
+                <div class="work-box w-disable">
+                    <div class="work-content">
+                        <h3 class="work-title">Data Dashboard Healthcare</h3>
+                        <p class="work-description">
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. has been the industry's standard.
+                        </p>
+                        <ul class="work-languages-list">
+                            <li class="work-language">HTML</li>
+                            <li class="work-language">Bootstrap</li>
+                            <li class="work-language">Ruby</li>
+                        </ul>
+                        <a href="#" class="btn see-work">See project</a>
+                    </div>
```

Comment:
- [ ] kindly consider making sure that when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width mentioned to meet the below requirement
> Lastly, when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.

Reply:
- [ ] The hover effect of the projects is not working yet. Please implement this feature.
![hover](https://user-images.githubusercontent.com/99656687/236414884-ce2ec2e7-af94-410a-981a-d5e6344c5a77.png)